packages/contracts/contracts/plugin/IPluginRepo.sol

```diff
@@ -6,12 +6,8 @@ pragma solidity 0.8.10;
 /// @author Aragon Association - 2022
 /// @notice The interface required for a plugin repository.
 interface IPluginRepo {
-    /// @notice Creates a new version with contract `_pluginSetupAddress` and content `@fromHex(_contentURI)`.
-    /// @param _semanticVersion The semantic version number for the version to be created.
-    /// @param _pluginSetupAddress The address of the plugin setup contract.
-    /// @param _contentURI External URI where the plugin metadata and subsequent resources can be fetched from
     function createVersion(
-        uint16[3] memory _semanticVersion,
+        uint8 _releaseId,
```

Comment:
The version is the combination of `release` and `build` numbers. If I name a `createRelease`, it should only create `release`, but since it creates both, the name is fine.

added the natspec. Feel free to resolve it if you want or let me know .